src/api-service/__app__/onefuzzlib/events.py

```diff
@@ -5,34 +5,26 @@
 
 import json
 import logging
-from queue import Empty, Queue
 from typing import Optional
 
 from onefuzztypes.events import Event, EventMessage, EventType, get_event_type
 from onefuzztypes.models import UserInfo
 from pydantic import BaseModel
 
 from .azure.creds import get_instance_id, get_instance_name
+from .azure.queue import send_message
+from .azure.storage import StorageType
 from .webhooks import Webhook
 
-EVENTS: Queue = Queue()
 
+def queue_signalr_event(event_message: EventMessage) -> None:
+    event = json.loads(event_message.json())
+    message = json.dumps({"target": "events", "arguments": [event]}).encode()
```

Comment:
how about defining a type for the envelope to reduce the number of conversion ?
``` python 
class SignalREvent(BaseModel):
    target: str
    arguments: List[EventMessage]
```

Reply:
Added.